index.js

```diff
@@ -206,6 +206,20 @@ function is(x) {
                 return false
             }
             return currYear - parseInt(x) == 13 
+        },
+        canSpell: {
+          thirteen: function(){
+            return x.toLowerCase().includes("t","h","i","r","t","e","e","n");
+          }
+        },
+        anagramOf:{
+          thirteen:function(){
+            if(x.toLowerCase().split('').sort().join('').trim() == "thirteen".split('').sort().join('').trim()){
```

Comment:
you can just:

    anagramOf:{
            thirteen:function(){
               return x.toLowerCase().split('').sort().join('').trim() == "thirteen".split('').sort().join('').trim();
             }
     }